weed/mount/filehandle_map.go

```diff
@@ -49,7 +50,9 @@ func (i *FileHandleToInode) AcquireFileHandle(wfs *WFS, inode uint64, entry *fil
 	} else {
 		fh.counter++
 	}
-	fh.entry = entry
+	if fh.entry != entry {
```

Comment:
I do not think this makes any difference though.

Reply:
I think I have a better solution now